nav2_dwb_controller/dwb_plugins/include/dwb_plugins/kinematic_parameters.hpp

```diff
@@ -80,9 +80,13 @@ struct KinematicParameters
   double min_vel_y_{0};
   double max_vel_x_{0};
   double max_vel_y_{0};
+  double entire_max_vel_x_{0};
```

Comment:
I'm not sure `entire_` is the right answer either.... `default_` I agree wasn't great. `base_`? `full_`?

Reply:
`_full` sounds good. I will update it in next commit

Reply:
Changed in https://github.com/ros-planning/navigation2/commit/34e694dbee7af9f805d2a7038b292466e44dc104